ports/usd/portfile.cmake

```
@@ -10,50 +10,37 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO PixarAnimationStudios/OpenUSD
     REF "v${USD_VERSION}"
-    SHA512 23f5a40c67ce0566cbb61a60a8d14ebb9fde5de0fb4921031f4d4eeb3a5a86d624955840f9f49ee36f32abc48e869d4910073189716e73ba997bb80f1e781d9b
+    SHA512 66bf75486f09dce7085c5c80cc3e005c02c1dfbbfbd02c8e2e8fd9498030ca49cd6bc0a9b9db930d0559c745cf77ebaf822018de4bf798cce84fdf0aa89f0d84
     HEAD_REF release
     PATCHES
-        001-fix_rename_find_package_to_find_dependency.patch # See PixarAnimationStudios/OpenUSD#3205
-        002-vcpkg_find_tbb.patch # See PixarAnimationStudios/OpenUSD#3207
         003-fix-dep.patch
         004-fix_cmake_package.patch
         007-fix_cmake_hgi_interop.patch
         008-fix_clang8_compiler_error.patch
         009-vcpkg_install_folder_conventions.patch
         010-cmake_export_plugin_as_modules.patch
-        011-TBB-2022.patch
-        012-fix-find-vulkan.patch
+        013-openimageio-3.patch
 )
 
-# Changes accompanying 006-vcpkg_find_spirv-reflect.patch
-vcpkg_replace_string("${SOURCE_PATH}/pxr/imaging/hgiVulkan/shaderCompiler.cpp"
-    [[#include "pxr/imaging/hgiVulkan/spirv_reflect.h"]]
-    [[#include <spirv_reflect.h>]]
-)
+# Changes accompanying 003-fix-dep.patch
 file(REMOVE
+    "${SOURCE_PATH}/cmake/modules/FindOpenColorIO.cmake"
     "${SOURCE_PATH}/pxr/imaging/hgiVulkan/spirv_reflect.cpp"
     "${SOURCE_PATH}/pxr/imaging/hgiVulkan/spirv_reflect.h"
+    "${SOURCE_PATH}/pxr/imaging/hgiVulkan/vk_mem_alloc.cpp"
+    "${SOURCE_PATH}/pxr/imaging/hgiVulkan/vk_mem_alloc.h"
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     FEATURES
-        materialx      PXR_ENABLE_MATERIALX_SUPPORT
-        metal          PXR_ENABLE_METAL_SUPPORT
         openimageio    PXR_BUILD_OPENIMAGEIO_PLUGIN
+        materialx      PXR_ENABLE_MATERIALX_SUPPORT
+        imaging        PXR_BUILD_IMAGING
+        imaging        PXR_BUILD_USD_IMAGING
+        imaging        PXR_ENABLE_GL_SUPPORT
```

```suggestion
        imaging        PXR_BUILD_IMAGING
        imaging        PXR_BUILD_USD_IMAGING
        imaging        PXR_ENABLE_GL_SUPPORT
        materialx      PXR_ENABLE_MATERIALX_SUPPORT
        openimageio    PXR_BUILD_OPENIMAGEIO_PLUGIN
```
Please sort these features alphabetically.